clients/web/test/spec/userSpec.js

```diff
@@ -96,6 +96,16 @@ describe('Create an admin and non-admin user', function () {
         }, 'front page to display');
     });
 
+    it('check the user count with an admin user', function () {
+        girderTest.login('admin', 'Admin', 'Admin', 'adminpassword!')();
+        girderTest.goToUsersPage()();
+        runs(function () {
```

Comment:
I wondered if it's better to have separate tests, than put this in the line 93 because it's two different tests